src/modules/app/createNavigateToCategory.js

```diff
@@ -19,13 +21,11 @@ export default (
       key,
       onRouteClose: () => dispatch({type: 'CLEAR_CATEGORY', params: {key}})
     },
-    // Use key when navigating to Categories. Else a new route is not pushed
-    key: routeName === 'Categories' ? key : undefined /* Explanation for "key":
-                                                         https://reactnavigation.org/docs/en/navigation-key.html */
+    key
```

Comment:
Are you sure the navigator is still working with this? I had the problem that it was sometimes not pushing a new screen when it should I think.

Reply:
I didn't notice any problems, could you describe where/when problems occured? Or maybe try it yourself?

Reply:
Apart from that, I don't see a reason why navigation to the Dashboard route wouldn't work if you supply a key.

Reply:
What is not working though is navigating back to the landing screen.

Reply:
That is expected behavior. The landing screen should only be displayed once in the future.

Reply:
I think I did it like this because of the behavior you mention in this PR: https://github.com/Integreat/integreat-react-native-app/pull/57#issuecomment-491813723

Apart from that I don't have clue.

Reply:
So there will be no possibility to change your location afterwards? Apart from that, is my solution by just not throwing errors in routeCleaner? Everything works like it should now.

Reply:
Have a look at my PR for pullToRefresh (https://github.com/Integreat/integreat-react-native-app/pull/59) I've fixed it there.

Reply:
Looks good, I'll wait until your PR is merged.